### PR TITLE
Language: use horizontal ellipsis instead of three dots or two dots

### PR DESCRIPTION
**Pull request by BKPepe**
I'm not a native English speaker, but I think two dots are wrong, what I have been able to find.
Also, I have replaced three dots (...) with the [horizontal ellipsis](https://en.wikipedia.org/wiki/Ellipsis#Computer_representations) (…), which is unicode character.

Sources:
https://english.stackexchange.com/questions/60452/usage-of-double-dots-is-it-formal
https://english.stackexchange.com/questions/443527/can-a-sentence-be-ended-with-two-dots

### CODE REVIEW

**Comment by Eldenroot** [comment]
I agree, looks better

File: inc/languages/english/admin/forum_attachments.lang.php [hunk 1]
```diff
@@ -72,11 +72,11 @@
 $l['step1of2_line1'] = "Please wait, the file system is currently being scanned for orphaned attachments.";
 $l['step2of2_line1'] = "Please wait, the database is currently being scanned for orphaned attachments.";
 $l['step_line2'] = "You'll automatically be redirected to the next step once this process is complete.";
-$l['scanning'] = 'Scanning..';
+$l['scanning'] = 'Scanning…';
```
Comment (effone) [inline inc/languages/english/admin/forum_attachments.lang.php]: Why its 2 here?

Comment (BKPepe) [reply inc/languages/english/admin/forum_attachments.lang.php]: Hard to tell. Maybe its some mistake, who left it there?

Comment (BKPepe) [reply inc/languages/english/admin/forum_attachments.lang.php]: I looked at it and it was added by this commit to language files:
https://github.com/mybb/mybb/commit/9dc1d0a73fae4e2ca0775925e245c500bbd01dfa#diff-bd2eb23b594576096e0256c0f4d8d62d ( https://github.com/mybb/mybb/issues/616) from this commit https://github.com/MyBBIr/mybb/commit/44de55595b5b1fc4014d9b80e6aa9b0017ae93bb

This was there for a long time.

**Comment by euantorano** [comment]
I believe this should use the HTML entity for compatibility, same as we use `&amp;` usually. The entity for this character seems to be `&hellip;`.

**Comment by euantorano** [comment]
Superseded by #3650 which uses the HTML entity.

**Pull request closed without merging**